editor/package.json

```diff
@@ -194,6 +194,10 @@
   },
   "devDependencies": {
     "@babel/cli": "7.8.4",
+    "@babel/core": "7.14.6",
```

Comment:
Is it worth updating all of the babel dependencies in this PR?

Reply:
maybe in a follow up PR?